QEfficient/transformers/models/modeling_auto.py

```diff
@@ -211,7 +211,6 @@ def export(self) -> str:
         Returns:
             :str: Path of the generated ``ONNX`` graph.
         """
-        assert self.is_transformed, "Please first run transform on the QEFFAutoModelForCausalLM object"
```

Comment:
Hey Amit, I removed it on onkar's review since its already being checked in export

- https://github.com/quic/efficient-transformers/pull/98#discussion_r1761226144

- https://github.com/quic/efficient-transformers/blob/b8970a4c2130deaa29bb274e40166493abdd246b/QEfficient/exporter/export_hf_to_cloud_ai_100.py#L180

Reply:
Okay, makes sense.